ui/src/lib/components/ImageFinder.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+    import Form from './Form.svelte';
+    import ResultsList from './ResultsList.svelte';
+    import { sendRequest } from '../utils/request.utils';
+    import type { ImagesRequest, ImagesResponse } from '../utils/request.utils';
+    import { Shadow } from 'svelte-loading-spinners';
+
+    $: results = [];
+    $: searching = false;
+
+    const createRequest = (formData): ImagesRequest => {
+        const request = {
+            path: formData.path,
+            moduleConfig: []
+        };
+
+        if(formData.metadata.active){
+            request.moduleConfig.push({
+                name: "metadata",
+                dateAfter: formData.metadata.createdAfter ?
+                    new Date(formData.metadata.createdAfter) : undefined,
+                dateBefore: formData.metadata.createdBefore ?
+                    new Date(formData.metadata.createdBefore) : undefined
+            });
+        };
+
+        return request;
+    }
+
+    const handleResponse = (response: ImagesResponse) => {
+        console.log("Response", response);
+        const items = [];
+        for(const item of response.pictures){
+            items.push({
+                index: items.length + 1,
+                url: item.split('\\').join('/'),
+                size: 'NA',
+                dimensions: 'NA',
+                name: item.replace(/^.*[\\\/]/, ''),
```

Comment:
Does the property name stand for the name of the file? If so then `.` shouldn't be removed from that name.

Reply:
`.` is not removed. It is symbol fot any char in regex.

Reply:
So this regex query replaces all '\' '/' signs with an empty string?

Reply:
replaces all chars ending with `/` or `\` with empty string
so `/dir1/dir2/file.jpg` -> `file.jpg`

Reply:
```suggestion
                name: item.replace(/^.*[\\\/]/, ''), // removes directory path from the file name
```